CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.1
+
+_Dec 17, 2021_
+
+A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🖨 Print export has a cleaner breaking page
+- 🧑‍💻 Add `pinnable` option
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### Changes
+
+- [DataGridPro] Add `pinnable` option (#3425) @m4theushw
+- [DataGridPro] Avoid filtering columns if no column is pinned (#3438) @m4theushw
+- [DataGrid] Avoid page break inside a row in the Print Export (#3302) @flaviendelangle
+- [DataGrid] Fix `GridEditDateCell` to handle `editRowsModel` correctly (#3267) @alexfauquette
+- [DataGrid] Refactor keyboard/click event management (#3275) @alexfauquette
+- [DataGrid] Fire change event when the state changes, not when firing the prop change callback in control mode (#3388) @flaviendelangle
+- [DataGrid] Unsubscribe event listeners registered in uncommitted renders (#3310) @m4theushw
+
+### Core
+
+- [core] Rework state update methods and deprecate `useGridApi` + `useGridState` (#3325) @flaviendelangle
+- [core] Add sections to some of the feature hooks (#3391) @flaviendelangle
+- [core] Generate exports snapshot for both `x-data-grid` and `x-data-grid-pro` packages (#3427) @flaviendelangle
+- [core] Remove 'x-data-grid' folder from DataGridPro bundle (#3394) @m4theushw
+- [core] Add link to OpenCollective (#3392) @oliviertassinari
+
+### Docs
+
+- [docs] Improve pagination documentation page (#3424) @flaviendelangle
+- [docs] Include @mui/x-data-grid as dependency in the CodeSandbox (#3396) @m4theushw
+- [docs] Stop using typedoc to generate the API documentation (#3320) @flaviendelangle
+- [docs] Remove column pinning from "Upcoming features" (#3443) @alexfauquette
+
+### Translation
+
+A big thanks to @sebastianfrey, @ColdAtNight and @jaapjr for improving the translation
```

Comment:
The PRs with contributions to the translations should be listed inside "Changes".